messaging/src/main/java/org/axonframework/serialization/json/JacksonSerializer.java

```diff
@@ -120,14 +119,16 @@ protected JacksonSerializer(Builder builder) {
     }
 
     /**
-     * Registers converters with the given {@code converter} which depend on the actual contents of the
-     * serialized for to represent a JSON format.
+     * Registers converters with the given {@code converter} which depend on the actual contents of the serialized for
```

Comment:
```suggestion
     * Registers converters with the given {@code converter} which depend on the actual contents of the serialized form
```
not sure `form` was the intended word here